Base/Python/slicer/parameterNodeWrapper/validators.py

```diff
@@ -132,8 +133,14 @@ class Choice(Validator):
     Validates that any input value is in the list of valid choices.
     """
 
-    def __init__(self, choices):
-        self.choices = choices
+    def __init__(self, *args):
```

Comment:
The input arguments are no longer self-explaining, therefore need to document what the method takes as input. Adding test or at least examples would be nice, too.

Reply:
I would prefer the caller have to do this conversion; too much magic going on to end up at the final list of choices.

For example:

```python
>>> Choice("Foo", "Bar")
Choice(['F', 'o', 'o', 'B', 'a', 'r'])
```

But intuitively I'd expect `Choice(['Foo', 'Bar'])`.

I don't think it's too bad to require the caller to flatten the choices before passing, and then `Choice`'s only argument is always the sequence of choices to use.

```python
def __init__(self, choices: Iterable):
    self.choices = list(choices)
```

Reply:
So, in other words, back out this change and keep the original functionality?

I missed the implication of using strings like in your `Choice("Foo", "Bar")` example, so I wouldn't necessarily be opposed to backing out the change.

The change was supposed to make things a little easier, not more confusing.

Reply:
Removed this commit to go back to old functionality.